vent/io/devices/pins.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, pin, pig=None):
     @pigpio_command
     def mode(self) -> str:
         """ The currently active pigpio mode of the pin."""
-        return dict(map(reversed, self._PIGPIO_MODES.items()))[self.pig.get_mode(self.pin)]
+        return {value: key for key, value in self._PIGPIO_MODES.items()}[self.pig.get_mode(self.pin)]

```

Comment:
make the inverted dict as a class attr instead of making each time in the property for clarity

Reply:
...fine